be/src/column/column.h

```diff
@@ -37,8 +38,10 @@ struct TypeDescriptor;
 // Forward declaration
 class Datum;
 
-class Column {
+class Column : public Cow<Column> {
```

Comment:
how about implements Cow method in Column?

Reply:
I think separate `Cow` from `Column` is better for maintaince:  `Cow` is decoupled with `Column` by implements/design ideally, so we can implement/test/use them separately.